### ISSUE TITLE
Failed to Fetch when exporting to Persistence

### ISSUE THREAD

**Issue by Stellar-tickets** [Sifchain/Support-Tickets#403]
Failed to Fetch error when exporting to Persistence , cleared cache etc. does not work

<img width="561" alt="Screen_Shot_2022-07-30_at_12 48 35_AM" src="https://user-images.githubusercontent.com/67415638/181904716-f2a5d097-fa23-4903-87bd-1407f92eab0d.png">

**Comment by lufa23** [comment]
looking at the devtools i can see:

`Access to fetch at 'https://rpc-persistence.keplr.app/' from origin 'https://sifchain.akash.pro' has been blocked by CORS policy: Response to preflight request doesn't pass access control check: No 'Access-Control-Allow-Origin' header is present on the requested resource. If an opaque response serves your needs, set the request's mode to 'no-cors' to fetch the resource with CORS disabled.`

Is this something @khdegraaf can help out?

<img width="709" alt="Screenshot 2022-07-30 at 11 41 03" src="https://user-images.githubusercontent.com/67415638/181904752-7e14918f-f614-4ef6-92ad-6bc3a525119b.png">

**Comment by Stellar-tickets** [comment]
lufa23: hey we found the error, let us get back to you on this

**Comment by Stellar-tickets** [comment]
lufa23: <@&961371260003172402>

**Comment by lufa23** [comment]
Looks like a FE issue, assigning the ticket to @bksifchain

**Comment by Stellar-tickets** [comment]
sif_baldr: Hi <@299720068059627524>. Are you using a Ledger device?

**Comment by Stellar-tickets** [comment]
sif_baldr: Just saw your response to another issue. Ok. We are looking into this issue.

**Comment by bksifchain** [comment]
Looks like a proxy may have been added to the configs and the FE never updated the endpoint. Will update and retest shortly.

@gzukel - https://github.com/Sifchain/proxies/blob/master/config.js#L97-L100

cc @alanrsoares

**Comment by Stellar-tickets** [comment]
sif_baldr: Hi <@299720068059627524> can you try again and let me know if there are any issues

**Comment by Stellar-tickets** [comment]
FrothyPint: I still get the same Failed to Fetch error.

**Comment by Stellar-tickets** [comment]
sif_baldr: Sorry. I thought the update was released but that's not the case. I will let you know when the fix is released.

**Comment by Stellar-tickets** [comment]
FrothyPint: Works after the update

**Comment by Stellar-tickets** [comment]
hungng: Yeah everything is fine now ?

**Comment by Stellar-tickets** [comment]
lufa23: !!close
